code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -781,3 +781,9 @@
 	id = /datum/reagent/consumable/red_queen
 	results = list(/datum/reagent/consumable/red_queen = 10)
 	required_reagents = list(/datum/reagent/consumable/tea = 6, /datum/reagent/mercury = 2, /datum/reagent/consumable/blackpepper = 1, /datum/reagent/growthserum = 1)
+
+/datum/chemical_reaction/mauna_loa
+	name = "Mauna Loa"
+	id = /datum/reagent/consumable/ethanol/mauna_loa
+	results = list(/datum/reagent/consumable/ethanol/mauna_loa = 25)
+	required_reagents = list(/datum/reagent/consumable/capsaicin = 10, /datum/reagent/consumable/ethanol/kahlua = 5, /datum/reagent/consumable/ethanol/bahama_mama = 10)
```

Comment:
```suggestion
	required_reagents = list(/datum/reagent/consumable/capsaicin = 2, /datum/reagent/consumable/ethanol/kahlua = 1, /datum/reagent/consumable/ethanol/bahama_mama = 2)
```
You should keep the ratios to their simplest form unless you're doing this for a particular reason (do also change the result to 5 units if you change this)

Reply:
I was thinking in button clicks from the Booze machine, not really raw numbers; this makes a lot more sense, and its definitely something I'll try to keep in mind for the future should I make any more drinks!